crypto/evp/keymgmt_lib.c

```diff
@@ -459,6 +459,12 @@ int evp_keymgmt_util_copy(EVP_PKEY *to, EVP_PKEY *from, int selection)
             evp_keymgmt_freedata(to_keymgmt, alloc_keydata);
             return 0;
         }
+    } else if (to_keymgmt == from->keymgmt && to_keymgmt->dup != NULL
+               && to_keydata == NULL
+               && selection == OSSL_KEYMGMT_SELECT_ALL) {
```

Comment:
Do we care what the selection is here?

Reply:
We do, but I am going to implement selective dup.